vendor/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go

```diff
@@ -634,6 +634,8 @@ func BuildStorageFactory(s *options.ServerRunOptions, apiResourceConfig *servers
 		return nil, fmt.Errorf("error in initializing storage factory: %s", err)
 	}
 
+	storageFactory.SetResourceEtcdPrefix(schema.GroupResource{Group: "apiregistration.k8s.io", Resource: "apiservices"}, "apiservices")
```

Comment:
what's the history of this? Did we ship an earlier version with a wrong etcd path?

Reply:
> what's the history of this? Did we ship an earlier version with a wrong etcd path?

Yes, in 3.6